components/tidb_query_vec_executors/src/index_scan_executor.rs

```diff
@@ -1020,7 +1015,7 @@ mod tests {
             Arc::new(EvalConfig::default()),
             columns_info,
             key_ranges,
-            1,
+            2,
```

Comment:
Why do we need to change this one?

Reply:
Because the last column of this table case is a int column and the second last one is a double. We want to ensure that we can decode a two-columns common handle (float, int), instead of just a normal int handle which is already cover by old index key/value handler arm.

Reply:
Sorry, I can't understand. Do you forget to change the construction from L999 to L1003?

Reply:
Yes. But the key layout is the same :rofl: